keyboards/projectcain/vault45/vault45.h

```diff
@@ -26,15 +26,52 @@
  * The second converts the arguments into a two-dimensional array which
  * represents the switch matrix.
  */
-#define LAYOUT( \
+
+#define LAYOUT_split1( \
```

Comment:
Similar deal here as on the Vault 35. Also adds a `LAYOUT_all` macro that supports programming every position on the PCB (though this layout cannot be built physically). We'll alias the exist `LAYOUT` macro to this, and then anyone who has a keymap out-of-tree for the existing macro will still be able to compile their keymap.

```suggestion
#define LAYOUT_all( \
    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, \
    K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110,       K112, \
    K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, \
          K301, K302,       K304, K305, K306, K307, K308,       K310, K311  \
) \
{ \
    { K000,  K001,  K002,  K003,  K004,  K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012 }, \
    { K100,  K101,  K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  KC_NO, K112 }, \
    { K200,  KC_NO, K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212 }, \
    { KC_NO, K301,  K302,  KC_NO, K304,  K305,  K306,  K307,  K308,  KC_NO, K310,  K311,  KC_NO }  \
}

#define LAYOUT_split_4space( \
```